src/index.html

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
+    Hello, world!
```

Comment:
It looks like you forget to add base markup to your code (`head`, `body` tags, etc)
Try to type `!` in empty file in your editor and press `tab` button and you will get base markup

Reply:
When I use tags (head, body), I have errors.
![image](https://user-images.githubusercontent.com/62118634/111786511-8e770900-88c6-11eb-89d4-45984ac2cb48.png)

Reply:
Read what do these errors say. They mean that you have the wrong space indentations before your tags. Just put 2 spaces before tags and errors will be gone